apps/admin/components/App.js

```diff
@@ -1,22 +1,27 @@
 import Head from 'next/head'
 import { withRouter } from 'next/router'
 import { fontFamilies } from '@project-r/styleguide'
+import { useMe } from '../lib/useMe'
 
-const App = ({ router, children }) => (
-  <main>
-    <Head>
-      <title>{router.pathname.replace('/', '')} – Admin</title>
-      <meta name='viewport' content='initial-scale=1.0, width=device-width' />
-      <style>{`
+const App = ({ router, children }) => {
+  useMe()
```

Comment:
Noted below, it's really not clear why this is used here (I know it's for the side effect of setting the Sentry user but this should be done explicitly)